include/master_element/MasterElementFactory.h

```diff
@@ -81,54 +61,27 @@ namespace nalu{
     std::map<stk::topology, MasterElement*>& meMap)
   {
     const stk::topology theTopo = AlgTraits::topo_;
-
+    //FIXME: ETI this
```

Comment:
What are you thinking in terms of ETI here?

Reply:
The method needs the definition of the master element to be available, e.g. Quad3DSCS.  To avoid header include ordering issues, we probably should just instantiate it for each ME.